app/src/main/res/values/dimens.xml

```diff
@@ -29,5 +29,6 @@
     <dimen name="margin_very_small">2dp</dimen>
     <dimen name="margin_large">10dp</dimen>
     <dimen name="message_textview_bottom_padding">3dp</dimen>
+    <dimen name="textview_layout_max_width">240dp</dimen>
```

Comment:
@chiragw15 The problem is that we are using custom views because of this match_parent is not working as expected as suggested by @iamareebjamal. I have tried tweaking the custom view as well but could not find a more probable solution for this.